src/components/Heading/Heading.tsx

```diff
@@ -64,23 +80,28 @@ export default function Heading(props: React.PropsWithChildren<HeadingProps>) {
 
   let content: string | React.ReactNode;
   if (props.children) {
-    content = url
-      ? React.createElement(
-          "a",
-          { href: url, className: urlClass },
-          props.children
-        )
-      : props.children;
+    content = url ? (
+      <Box as={ChakraLink} href={url} className={urlClass}>
+        {props.children}
+      </Box>
+    ) : (
+      props.children
+    );
   } else {
-    content = url
-      ? React.createElement("a", { href: url, className: urlClass }, text)
-      : text;
+    content = url ? (
+      <Box as={ChakraLink} href={url} className={urlClass}>
+        {text}
+      </Box>
+    ) : (
+      text
+    );
   }
```

Comment:
This might be slightly easier to read if it was something like
```
const contentToRender = props.children ? props.children : text;
content = url ? (
    <Box as={ChakraLink} href={url} className={urlClass}>
      {contentToRender}
    </Box>
  ) : (
    contentToRender
  );
```

Reply:
Yes, I like that.